builder/base/twig/kss_builder_base_twig.js

```diff
@@ -73,9 +73,10 @@ class KssBuilderBaseTwig extends KssBuilderBase {
       };
       this.options.namespace.forEach(namespace => {
         // namespace should be of the form "namespace:path";
-        let tokens = namespace.split(':', 2);
-        if (tokens[1]) {
-          this.namespaces[tokens[0]] = path.resolve(tokens[1]);
+        let namespacePart = namespace.substr(0, namespace.indexOf(':'));
+        let pathPart = namespace.substr(namespace.indexOf(':') + 1);
```

Comment:
```suggestion
        const pathPart = namespace.substr(namespace.indexOf(':') + 1);
```

Reply:
Same here. Generally there are not really constants but scoped variables.

Reply:
Yes, `const` are only scoped variable "unreasignable".